### ISSUE TITLE
Vim acting funny on latest build

### ISSUE THREAD

**Issue by Fedreg** [blinksh/blink#113]
My VIM sessions are not displaying properly.  Getting many random characters such as 

,0-1
, 22
, 9,8
etc  along the bottom of the screen.  It is intermittent and I can't recreate it regularly.  Sometimes it resolves itself after the screen sleeps; sometimes not.  Reconnecting usually fixes it.   Never have this issue when connecting through other devices or terminals.   

Ipad Air 2 latest ios.

**Comment by carloscabanero** [comment]
Thanks a lot for the report! 

I wonder if it might have something to do with the Caps as ESC configuration. Do you have any special mapping for the keys? Also, latest iOS means iOS 10?

Any other VIM users (@yury @goerz @panchoh ...) experiencing this?

**Comment by Fedreg** [comment]
Carlos, thanks for the note.  It actually was something in my .vimrc file.  I had started using someone elses vimrc for their Vue.js configurations and the issue had only popped up when using Blink.  However, I'm not a heavy a heavy Vim user and had never encountered the issue before while using Coda's terminal, GoCoEdit's terminal, or the shell from Chrome OS.  I was able to recreate it on Coda's terminal after using Vim a bit more heavily and doing more yank/put on the lines.  Happened much more frequently on blink though.  Could mosh have something to do with this???  Not really sure all the technical differences between Mosh and ssh.   I've gone back to my vimrc and no problems thus far.  Sorry to waste your time!  Just assumed it was blink since this had not popped before.  Will be more dilligent before next issue report!!!  Really enjoying blink!!    ......but liked the old logo better! Thanks for your help
